### PR TITLE
Kazoo 4701 4.0

### PR DESCRIPTION
**Pull request by jamesaimonetti**

### CODE REVIEW

File: core/kazoo_apps/src/kapps_call.erl [hunk 1]
```diff
@@ -665,10 +685,17 @@ callee_id_number(#kapps_call{callee_id_number=CIDNumber}) -> CIDNumber.
...
+                     ,request_user=to_e164(RequestUser)
                      ,request_realm=RequestRealm
                     }.
 
+-ifdef(TEST).
+to_e164(Number) -> Number.
+-else.
+to_e164(Number) ->
+    knm_converters:normalize(Number).
+-endif.
```
Comment (fenollp) [inline core/kazoo_apps/src/kapps_call.erl]: Why is this necessary?

Comment (jamesaimonetti) [inline core/kazoo_apps/src/kapps_call.erl]: at least with the wnm_util version, it queried the db to get the regex. We aren't testing to_e164 so skip that check
